src/json-ld.net/Core/JsonLdError.cs

```diff
@@ -11,14 +11,22 @@ public class JsonLdError : Exception
         private JsonLdError.Error type;
         internal JObject details = null;
 
-        public JsonLdError(JsonLdError.Error type, object detail) : base(detail == null ? 
-            string.Empty : detail.ToString())
+        public JsonLdError(JsonLdError.Error type, object detail, Exception innerException)
+            : base(detail == null ? string.Empty : detail.ToString(), innerException)
         {
             // TODO: pretty toString (e.g. print whole json objects)
             this.type = type;
         }
 
-        public JsonLdError(JsonLdError.Error type) : base(string.Empty)
+        public JsonLdError(JsonLdError.Error type, object detail)
+            : base(detail == null ? string.Empty : detail.ToString())
+        {
+            // TODO: pretty toString (e.g. print whole json objects)
```

Comment:
Can't we just use JsonConvert.Serialize ?

Reply:
I did not add the `TODO`, so I'd rather not do anything about that in this PR.

Reply:
Ah ok